tools/test_containers_compose

```diff
@@ -9,4 +9,13 @@ for i in webui worker; do
     sudo docker-compose up -d &&
     (docker-compose ps --services --filter status=stopped | grep "^[[:space:]]*$") || (docker-compose logs; sudo docker-compose ps; exit 1)
   ) || exit 1;
+
+  if [[ $i == "webui" ]]; then
+    (
+      docker-compose exec webui /usr/share/openqa/script/clone_job.pl --from https://openqa.opensuse.org --host http://nginx:9526 1738950 || echo "Error executing a job";
+      sleep 1;
```

Comment:
Shouldn't there be a better way?

Reply:
I agree

Reply:
I did a new approach, please check this new commit